### ISSUE TITLE
We should add map support

### ISSUE THREAD

**Issue by kelson42** [openzim/mwoffliner#288]
To have maps in Wikivoyage and Wikipedia ZIM files.

Depends on https://github.com/openzim/libzim/issues/136

**Comment by Jaifroid** [comment]
Could this be handled by the OS's maps app or mapping system? For Kiwix JS Windows, I transform the geo: links in the Wikivoyage English and Wikivoyage German ZIM files so that when clicked on, they open the Maps App with the location centred on the geo: co-ordinates and a location pin with the name of the feature. If the user has downloaded the country's offline map, then this works offline, otherwise, the app connects to the Internet to show the location. It works on Windows Mobile and on Windows desktop. However, I agree it would be great if Wikivoyage ZIMs had at least a snapshot of the map that is shown in the Web versions of many pages.

**Comment by kelson42** [comment]
@Jaifroid Yes, this is how it should work today and should continue to work in the future. But we want to be able to provide a better level of integration between map/text/picture contents.

**Comment by Popolechien** [comment]
I'm pushing this issue back up to know if there's been any progress on that front - either by lokoing for a tech solution to integrating maps, or by making sure that their absence is not so noticeable:

![Capture d’écran 2023-12-11 à 13 44 53](https://github.com/openzim/mwoffliner/assets/21675885/16e7e65d-deb3-4e63-87c2-49ad8cd8475e)

![Capture d’écran 2023-12-11 à 13 45 04](https://github.com/openzim/mwoffliner/assets/21675885/f9eddc9c-52ff-4773-be0e-8da35e6749a8)

**Comment by kelson42** [comment]
@Popolechien I think your example does not really fit to what I meant first, I have open [a dedicated issue](https://github.com/openzim/mwoffliner/issues/2434) for that. I have rewritten the issue description to be clearer about the problem.
